jbehave-core/src/main/java/org/jbehave/core/reporters/StoryReporter.java

```diff
@@ -97,5 +101,6 @@ public interface StoryReporter {
 
     void dryRun();
 
+    @Deprecated
     void pendingMethods(List<String> methods);
```

Comment:
it looks like you removed the reporting of this hook at all, but it should still be invoked even though it's deprecated